vault-plugins/flant_iam/backend/path_user.go

```diff
@@ -423,7 +424,11 @@ func (b *userBackend) handleExistence() framework.ExistenceFunc {
 func (b *userBackend) handleCreate(expectID bool) framework.OperationFunc {
 	return func(ctx context.Context, req *logical.Request, data *framework.FieldData) (*logical.Response, error) {
 		b.Logger().Debug("create user", "path", req.Path)
-		id := getCreationID(expectID, data)
+		id, err := backentutils.GetCreationID(expectID, data)
+		if err != nil {
+			return backentutils.ResponseErrMessage(req, err.Error(), http.StatusBadRequest)
+		}
```

Comment:
If `GetCreationID` returns an error from `uuid.New()`, it is not 'bad request', it's 'internal server error' (multiple places)

Reply:
no, uuid.New() can't produce error, error can happen in case request doesn't provide 'uuid' param => bad_request